tests_openshift/database/test_models.py

```diff
@@ -793,18 +804,17 @@ def test_merged_runs(clean_before_and_after, few_runs):
         merged_run = PipelineModel.get_merged_run(run_id)
         srpm_build_id = merged_run.srpm_build_id
 
-        # for different_pr (i=1) there are Copr builds twice, since the second
-        # run of TFT from the same Copr build produces new row with same SRPM
-        # and Copr IDs, but different Testing Farm IDs
-        # ^ handled in API by iterating over set of IDs instead of list
-        assert len(merged_run.copr_build_id) == 2 * i
+        # Since the introduction of build groups, the builds are grouped
+        assert len(merged_run.copr_build_group_id) == 1
```

Comment:
I had to change this test because now the pipelines creation/copying works differently. Do you think that this is fine? @mfocko I see you were the author of the change, I'd appreciate your opinion

Reply:
I'd say it's fine.